setup/prompt.ts

```diff
@@ -1,17 +1,23 @@
-const { prompt } = require("enquirer");
+import { prompt } from "enquirer";
 
-export const askSessionToken = async () => {
-  const { sessionToken } = await prompt({
+const singlePrompt = ({ message }: { message: string }) =>
+  prompt({
     type: "input",
-    name: "sessionToken",
-    message: "Insert a valid session token",
+    name: "value",
+    message,
     validate(value: string /*, state, item, index*/) {
       if (!value) {
-        return 'required';
+        return "required";
       }
       return true;
     }
-  });
+  }).then((e: Partial<{ value: string }>) => e.value || "");
 
-  return sessionToken;
-};
+export const askSessionToken = () =>
+  singlePrompt({ message: "Insert a valid session token" });
+
+export const askIOBackendHost = () =>
+  singlePrompt({ message: "Insert the IO Backend host" });
+
+export const askIOBackendBasePath = () =>
+  singlePrompt({ message: "Insert the IO Backend base path" });
```

Comment:
there's no need to alias `singlePrompt()`, utility methods must add value, just use it inline when needed

Reply:
Aliases avoid prompt details to leak into the `index.ts` ("main") module, which don't necessary need to know details such message, validation or even the source of the input. 
(it's just a coincidence here that all three inputs share the same configuration).

Although this conflicts with the principle of "make the code explicit", it stands for the principle of "keep each code unit at the same level of abstraction".

Reply:
that's exactly the main module that should know about message prompts. exactly where getopts belong.

Reply:
ok